monitoring/values-prom-operator.yaml

```diff
@@ -196,6 +196,9 @@ grafana:
         format: json
     dashboards:
       default_home_dashboard_path: /tmp/dashboards/viya-welcome-dashboard.json
+    initChownData:
```

Comment:
Hmmm, still concerned that the indentation might not be exactly right.  In this file,  it appears that the **intiChownData:** line lines up with the **analytics:**, **log:,** **"log.console":** and **dashboards:** lines...which all seem to be indented two spaces deeper than the **image:** and **"grafana.ini":** lines.  I suspect those should all line up together.  As they are now, it makes them all appear to be "children" under the **"grafana.ini":** line.  Note that they lineup with the **image.tag** line.